.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: python
 python:
   - 3.5
+services:
+  - postgresql
 env:
   - TRAVIS_NODE_VERSION="4.4"
 install:
   - pip install -r requirements.txt
-  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
-  - npm install
```

Comment:
Is there a reason node/npm were being installed here? It's not in our README as a requirement so I've removed it for now...